### ISSUE TITLE
Document branch coverage

### ISSUE THREAD

**Issue by eliu1217** [eliu1217/assignment3-albumentations#9]
Document and assess the manually implemented branch coverage for 5 functions with high CC.

**Comment by danielwis** [comment]
Is this required (i.e. documenting the manually implemented coverage? I couldn't find anything about in on Canvas.

**Comment by eliu1217** [comment]
Yeah just checked grading criteria and it seems that documentation is not needed. Can be covered by the other issue "Improve branch coverage".
